src/util/webauthn.js

```diff
@@ -93,5 +93,8 @@ export default {
       isAvailable = await PublicKeyCredential.isConditionalMediationAvailable();
     }
     return isAvailable;
-  }
+  },
+  isRelyingPartyIdMismatchError: function(error) {
+    return error?.name === 'SecurityError' && error?.code === 18;
```

Comment:
Is 18 the code number that the WebAuthn browser library returns, basically?

Reply:
yup, that's the error code thrown by the browser